go.mod

```diff
@@ -6,7 +6,7 @@ toolchain go1.23.4
 
 require (
 	github.com/equinor/radix-common v1.9.7
-	github.com/equinor/radix-operator v1.70.2
+	github.com/equinor/radix-operator v1.70.3-0.20250212135212-cf78d8afc50b
```

Comment:
Is this the final/correct version?

Reply:
No, this will be fixed when radix+operator is updated